Fix - Group Permission Restriction

ehmadzubair opened the pull request:
Fixes #78 

We'll still need to add a template for this 403 page to look 'decent'. Not spending time on it right now.

istride commented:
@ehmadzubair my understanding is that you will be adding some test cases to this PR

ehmadzubair commented:
Yes, that is correct. Right after I complete comments

ehmadzubair commented:
@istride I have updated this PR with the Test Cases

ehmadzubair commented on iogt/templates/header.html:
```diff
@@ -36,7 +36,7 @@
         <div class="header-middle">
             <a class="logo-small" href='/'>
-                <img src="{% static "/images/iogt-logo-two-lines.png" %}"/>
+                <img src="{% static "images/iogt-logo-two-lines.png" %}"/>
```
This was causing the test suite to not run.

ehmadzubair commented:
@isedwards If you can review/merge this please.

istride reviewed (changes requested):
@ehmadzubair would you add some instructions to the README about how to run the tests. Perhaps add a target to the Makefile.

I'm having trouble running the tests. Firstly, some issue with static files, for which, I ran `./manage.py collectstatic` to resolve. Secondly, I have an issue with the Django Debug Toolbar, which I have not found a fix for, as follows:

```
django.urls.exceptions.NoReverseMatch: 'djdt' is not a registered namespace
```

ehmadzubair commented:
@istride I've added separate settings for running tests and both the issues have been resolved (not snoozed). I've also added the readme steps to run the tests
